<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JuliaFractalWidget</class>
 <widget class="QWidget" name="JuliaFractalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Iterations:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="iterationsInput">
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setIterationsButton">
     <property name="text">
      <string>Set Iterations</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Minimum C-Real:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="minCRealInput">
     <property name="text">
      <string>-1.2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Maximum C-Real:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxCRealInput">
     <property name="text">
      <string>1.2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Minimum C-Imaginary:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="minCImagInput">
     <property name="text">
      <string>-1.2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Maximum C-Imaginary:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxCImagInput">
     <property name="text">
      <string>1.2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>K-Real</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="kRealInput">
     <property name="text">
      <string>0.285</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>K-Imag</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="kImagInput">
     <property name="text">
      <string>0.01</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="generateButton">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetZoomButton">
     <property name="text">
      <string>Reset Zoom</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
